documentatie/use-cases/GSB-gebruikersdoelen.md

```diff
@@ -107,33 +107,29 @@ __uitbreidingen__:
 - OSV richt de server ook gelijk als client in. Willen wij dit ook?
 - Afzenderverificatie ontbreekt nog, want nog geen beslissing over oplossing.
 
-## GSB richt de applicatie in
+
+## De coördinator maakt de stembureaus aan.
 
 __niveau:__ gebruikersdoel, zee, 🌊
 
 ### Hoofdscenario en uitbreidingen
 
-__trigger:__
-
-- [GSB installeert de applicatie.](#gsb-installeert-de-applicatie)
-- Het GSB maakt de kandidatenlijst beschikbaar.
-- De Kiesraad maakt de verkiezingsdefinitie beschikbaar.
-
 __hoofdscenario__:
 
-1. De beheerder leest de verkiezingsdefinitie in.
-2. De beheerder leest de kandidatenlijst in.
-3. De beheerder maakt de stembureaus aan.
-4. De beheerder maakt de gebruikers aan.
+1. De coördinator importeert de lijst van stembureaus.
+2. De applicatie controleert dat de stembureaus aan de validaties voldoen.
```

Comment:
```suggestion
2. De applicatie stelt vast dat de stembureaus aan de validaties voldoen.
```

Reply:
Heb alle "controleert"s in de use cases vervangen door "stelt vast"s.

Reply:
done